Contribute/Code-samples/index.md

```diff
@@ -0,0 +1,131 @@
+---
+meta.Title: "Code samples - Best practices"
+meta.Description: "We highly recommend supporting your documentation articles with code samples. Make sure the samples follow our best practices outlined in this article."
+---
+
+# Code samples
+
+The articles in the Umbraco Documentation can in most cases benefit from relevant code samples to support the written text.
+
+In this article you will find some best practices which outline how we recommend formatting and using code samples, and you will also find definitions and examples of the most used types of code samples in the Umbraco Documentation.
+
+## Best practices
+
+In order to ensure the quality and consistence of all the various code samples in the Umbraco Documentation, we have outlined a set of best practices that we highly recommend that you follow, when adding code snippets to articles.
+
+* Add a clear description
+* Use code comments
+* Use real-life samples
+* Add correct syntax highlighting
+* Add only compilable samples
```

Comment:
```suggestion
* Add only compilable samples
* Include namespace 'Using' statements
```

Reply:
I can see you've mentioned the Using Statements in the section about compilable samples - should it have it's own heading and section?
Otherwise, I don't think it should be a bullet on this list - but be part of the "Add only compilable samples" bullet?
What do you think?

Reply:
Agree, I went down linearly... so though it was a big enough deal to have it's own point, then read the compilable text - and really 'it's part of that' - so remove the bullet? 
or
* Add only complete compilable samples (including Using statements) 

happy either way, but yes, shouldn't have a bullet without a heading and vice versa

Reply:
I like adding it to the existing bullet! I'll take of that.

Reply:
I've updated the bullet and heading to match 💪